stellarphot/settings/tests/test_custom_widgets.py

```diff
@@ -7,6 +7,7 @@
     Observatory,
     PassbandMap,
     SavedSettings,
+    settings_files,
```

Comment:
What is this change doing?  There is no `settings_files` class, just the entire module (which contains your other settings classes you are importing here).

OK, on reviewing the code, it looks like you import `settings_files` directly to allow using of mocked versions of the attributes it contains.  That said, in the code it looks like `settings_file` is just an instantiation of the `SavedSettings` class so I am still troubled by this import and the matching import in `test_settings_file.py`.

Reply:
@JuanCab  -- that is indeed confusing (I tried a bunch of variants of the import before getting something that worked). I added some more explanation [here](https://github.com/feder-observatory/stellarphot/pull/372/files#diff-b03c6eb9430bafc1d0960b6c6f312ba8afd1d7734bc60e7bd2b3b4e72bc03d58R85-R91). Can you please take a look and let me know if that helps clear it up?

Reply:
Thanks for the clarification, I have a better idea why although still seems a little like voodoo. :)

Reply:
Yeah, it is definitely voodoo!